epidose/device/wps_scanner_d.sh

```diff
@@ -147,7 +147,7 @@ connect_to_saved_network()
   fi
 
   # Get $1 network's id, list_networks checks the /etc/wpa_supplicant/wpa_supplicant.conf file
-  network_id=$(wpa_cli list_networks | awk '$2 ~ /^"$1"$/ {print $1}')
+  network_id=$(wpa_cli list_networks | awk -v network_name="$1" '$2 ~ network_name {print $1}')
```

Comment:
Nice!  Why not ==?

Reply:
I had no clue. This is how I found it in a forum.
I change and test it and then re-submit if you want me to.

Reply:
Yes please.  Otherwise you catch networks that aren't correct, e.g. eduroamtest.  Also, I see why the previous failed.  It should be `/^'"$1"'$/`.

Reply:
Ahhh, I see.
I have submitted a PR that addresses the requested changes.